### DISCUSSION TITLE
INI was not able to parse ru_ofh.++

### DISCUSSION THREAD

**Discussion by wei-lou** [srsran/srsRAN_Project#377] [General Help] [answered]
Installed 23.10.1 from the "ppa:softwareradiosystems/srsran-project" repository on both VM and baremetal server running ubuntu 22.04 minimal server version. Every I am trying to run the gnb, it pop up this error.

$ sudo gnb -c srsRAN_Project/configs/gnb_ru_fuji_tdd_n48_40mhz_2x2_23101R.yml --help
INI was not able to parse ru_ofh.++
Run with --help for more information.

I can't collect the gnb.log since it hasn't really started yet.

**Comment by andrepuschmann** [comment, marked as answer]
This is likely a mismatch in configs or a formatting issue. Take a default config for that version and do your changes step-by-step until it parses the file correctly.

**Reply by wei-lou** [reply]
arrh, you are right. I was using the same yaml file from 23101 Test branch. After used the official 23.10.1 main branch yaml file. I was able to run gnb with all edition for my RU. It worked with the compiled code on my server. Thanks a lot for the help.

**Reply by andrepuschmann** [reply]
Glad it worked.
